data/plugins/findfile.lua

```diff
@@ -7,39 +7,64 @@ local config = require "core.config"
 local keymap = require "core.keymap"
 
 config.plugins.findfile = common.merge({
+  -- how many files from the project we store in a list before we stop
   file_limit = 20000,
-  max_search_time = 2.0
+  -- the maximum amount of time we spent gathering files before stopping
+  max_search_time = 2.0,
+  -- the amount of time we wait between loops of gathering files
+  interval = 1 / config.fps,
```

Comment:
Adding this here so it's not lost in Discord: should we just do this as quickly as possible, so with `interval = 0`?

Reply:
Sure.

Reply:
Done.